src/main/java/net/mindengine/galen/runner/GalenArguments.java

```diff
@@ -355,59 +333,31 @@ public boolean equals(Object obj) {
         if (!(obj instanceof GalenArguments)) {
             return false;
         }
-        GalenArguments rhs = (GalenArguments)obj;
-        return new EqualsBuilder()
-            .append(action, rhs.action)
-            .append(paths, rhs.paths)
-            .append(recursive, rhs.recursive)
-            .append(javascript, rhs.javascript)
-            .append(includedTags, rhs.includedTags)
-            .append(excludedTags, rhs.excludedTags)
-            .append(screenSize, rhs.screenSize)
-            .append(htmlReport, rhs.htmlReport)
-            .append(testngReport, rhs.testngReport)
-            .append(url, rhs.url)
-            .append(filter, rhs.filter)
-            .append(parallelSuites, rhs.parallelSuites)
-            .append(excludedTags, rhs.excludedTags)
-            .append(export, rhs.export)
-            .append(maxWidth, rhs.maxWidth)
-            .append(maxHeight, rhs.maxHeight)
-            .append(jsonReport, rhs.jsonReport)
-            .isEquals();
-    }
-    
+        final GalenArguments rhs = (GalenArguments) obj;
```

Comment:
I am also against this kind of refactoring. It doesn't make the code more readable when all of the things are moved in one line. It seems like this was generated with some kind of batch refactoring tool, is it?